extensions/blocks/tiled-gallery/view.scss

```diff
@@ -33,6 +34,12 @@ $tiled-gallery-max-column-count: 20;
 			display: flex;
 		}
 	}
+
+	@for $i from 1 through $tiled-gallery-max-rounded-corners {
```

Comment:
This will generate 20 lines of CSS like this. Figured it's lightweight enough addition when limited to 20 or so.

Alternatively (maybe in another PR) we have this in separate CSS file which we enqueue only when they have rounded corners, or generate this in PHP for the exact number as inline CSS.